content/docs/rfcs/34/README.md

```diff
@@ -86,6 +89,8 @@ message PeerExchangeRPC {
 The `enr` field contains a Waku ENR as specified in [31/WAKU2-ENR](/spec/31/).
 
 Requesters send a `PeerExchangeQuery` to a peer.
+When `cluster_id` is set and `shards` are specified, responder SHOULD provide peers belonging to cluster_id/shards pair.
```

Comment:
```suggestion
When `cluster_id` is set and `shards` are specified, responders SHOULD provide peers belonging to the specified cluster_id/shards pair.
```
Full sentences for RFCs, in other words it would be either `the responder` or `responders`.